dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -373,17 +373,21 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
             QueryHistoryExtendedFormat.ZIP_ENTRY_PREFIX,
             createExtendedQueryFromAccountUsage(arguments),
             QueryHistoryExtendedFormat.Header.class);
-    queryLogIntervals.forEach(interval -> addJdbcTask(out, interval, queryHistoryTask));
+    queryLogIntervals.forEach(interval -> addJdbcTask(out, interval, queryHistoryTask, true));
 
     List<TaskDescription> timeSeriesTasks = createTimeSeriesTasks(arguments);
     Duration duration = Duration.ofDays(1);
     ZonedIntervalIterableGenerator.forConnectorArguments(
             arguments, duration, IntervalExpander.createBasedOnDuration(duration))
-        .forEach(interval -> timeSeriesTasks.forEach(task -> addJdbcTask(out, interval, task)));
+        .forEach(
+            interval -> timeSeriesTasks.forEach(task -> addJdbcTask(out, interval, task, false)));
```

Comment:
These time series tasks are running with frequency of 1 day. So we should not consider these days as log dates and need to be filter out.